crypto/aes/asm/bsaes-armv7.pl

```diff
@@ -49,10 +48,7 @@
 #						<appro@openssl.org>
 
 # April-August 2013
-#
 # Add CBC, CTR and XTS subroutines, adapt for kernel use.
-#
-#					<ard.biesheuvel@linaro.org>
```

Comment:
"Ard Biesheuvel of Linaro"

Reply:
add "courtesy of Ard" since his name/affiliation is just 20 liens up.